src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1553,6 +1554,11 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
                 ->addTag('messenger.receiver', array('alias' => $name))
                 ->addTag('messenger.sender', array('alias' => $name))
             ;
+
+            if ('symfony://kernel.terminate' === $transport) {
+                $transportDefinition->addTag('kernel.event_listener', array('event' => KernelEvents::TERMINATE));
```

Comment:
Can't you take the event from what is after `symfony://` instead of hardcoding it?

Reply:
Does it make sense if we're only going to listen to that event?

Reply:
See comments after https://github.com/symfony/symfony/issues/28646#issuecomment-427994441.
I'm not convinced another use-case should be provided out-of-the-box by the Framework bundle.